src/sentry/sdk_updates.py

```diff
@@ -216,6 +216,13 @@ def get_new_state(self, old_state):
         'module_version_min': '5.0.0',
         'suggestion': EnableIntegrationSuggestion('tornado', 'https://docs.sentry.io/platforms/python/tornado/')
     },
+    {
+        'sdk_name': 'sentry.python',
+        'sdk_version_added': '0.10.0',
+        'module_name': 'redis',
+        'module_version_min': '0.0.0',
+        'suggestion': EnableIntegrationSuggestion('redis', 'https://docs.sentry.io/platforms/python/redis/')
```

Comment:
Redis or RQ? There's no https://docs.sentry.io/platforms/python/redis/ in our docs.

Reply:
Ah, yes. This depends on https://github.com/getsentry/sentry-docs/pull/1118